.circleci/config.yml

```diff
@@ -3,24 +3,15 @@ version: 2.1
 executors:
   nodejs:
     docker:
-      - image: circleci/node:10
+      - image: dannycoates/fxa-circleci
```

Comment:
This image should move somewhere else, but it's been useful for testing, and probably ok in the short term. IDK the best place for it. It's just a `Dockerfile`. It _could_ go in the fxa repo, but then would it have a ci step... 🤔

Reply:
A separate fxa repo with an auto-updated docker hub build?